pkg/brokerapi/service_instance.go

```diff
@@ -40,6 +40,20 @@ type CreateServiceInstanceRequest struct {
 	SpaceID           string                 `json:"space_guid,omitempty"`
 	Parameters        map[string]interface{} `json:"parameters,omitempty"`
 	AcceptsIncomplete bool                   `json:"accepts_incomplete,omitempty"`
+	ContextProfile    ContextProfile         `json:"context,omitempty"`
```

Comment:
do you have a nice refactor tool I can look into? I've had no luck historically.